servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile.glsl

```diff
@@ -178,8 +182,6 @@ void main() {
 	color_interp = color_attrib;
 #endif
 
-	bool is_multimesh = bool(instances.data[draw_call.instance_index].flags & INSTANCE_FLAGS_MULTIMESH);
-
```

Comment:
This is changing the way `is_multimesh` is detected in the s**cene_forward_movine.glsl** shader, but non mobile shaders will continue using the old way. Is there a reason to not unify criterias in different shaders?

Reply:
Yes, we will be changing how multimesh are handled in both shaders in 4.4. But before we do that, we need to merge https://github.com/godotengine/godot/pull/90400 otherwise we will make the shader compilation stutter much worse. 

This PR is a band aid fix for 4.3. It only targets Mobile since the Quest3 is the only device with this driver bug and the cost of adding a new spec_constant in the Forward+ shader is much higher. The way the Mobile renderer works right now, this change won't increase shader compilation stutter, so it is safe to make. In the Forward+ renderer this change could increase shader compilation stutter (its already a much bigger problem there) and since the Forward+ renderer can't be used on the Quest3, it doesn't really provide a benefit.